src/d3d12/d3d12_renderer.hpp

```diff
@@ -68,6 +69,8 @@ namespace wr
 		wr::CommandList* GetComputeCommandList(unsigned int num_allocators) final;
 		wr::CommandList* GetCopyCommandList(unsigned int num_allocators) final;
 		RenderTarget* GetRenderTarget(RenderTargetProperties properties) final;
+		d3d12::HeapResource* GetLightBuffer();
```

Comment:
Should be part of the scene graph.

Reply:
Not available yet - but, Koen is now working on a StructuredBuffer resource pool and it will be commited into this branch